src/hazelweb/keyboard/Key.re

```diff
@@ -31,7 +135,7 @@ let recognize = (evt: Js.t(Dom_html.keyboardEvent), r) =>
     let code = get_code(evt);
     String.equal(code, c);
   | Key(k) =>
-    let key = get_key(evt);
+    let key = without_option_key(get_key(evt));
```

Comment:
also probably only want to do this in situations where we are recognizing a key combination involving alt -- don't necessarily want to preclude allowing unicode character entry into the code directly

Reply:
Implemented this request in commit [`916730d`](https://github.com/hazelgrove/hazel/pull/523/commits/916730deefe0de97ee714cd363235a2bdba1c7c0).